conf/trainer/logger/sparsewandb.yaml

```diff
@@ -0,0 +1 @@
+_target_: lightning_transformers.core.loggers.WABLogger
```

Comment:
Is this logger a necessity? Could we add this in a separate PR? why is it needed for SparseML?

Reply:
SparseML calls special functions (_lambda_func, log_scalar, and log_scalars) to the logger. Why? It looks like they needed it to convert certain arguments to JSON-compatible format. Instead of making a wrapper for loggers, what they should have done instead is convert those arguments before feeding them into the logger. This makes it impossible to use other loggers directly without making an entirely new wrapper.  I'm thinking about opening a PR at the sparseml repository to get their code to talk normally to a logger.

To prevent the user from accidentally using wandb from default, I've included it as the default. Until someone (probably me) opens a PR at SparseML, this is unfortunately what we have to work with.

Reply:
Makes sense, thanks @mathemusician :)